File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/TabViewHolder.java
package org.fossasia.susi.ai.chat.adapters.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import org.fossasia.susi.ai.R;

import butterknife.BindView;
import butterknife.ButterKnife;

/*
*  A view for the individual item of the recyclerlist
* */

public class TabViewHolder extends RecyclerView.ViewHolder{

    @BindView(R.id.parentLayout)
    public RecyclerView view;

    public TabViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/TabViewHolder.kt
package org.fossasia.susi.ai.chat.adapters.viewholders

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import kotterknife.bindView
import org.fossasia.susi.ai.R

class TabViewHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    val view: androidx.recyclerview.widget.RecyclerView by bindView(R.id.parentLayout)
}
